### PR TITLE
Feature/duck

### PR DESCRIPTION
**Pull request by khuatanh**
please check code for me

### CODE REVIEW

**Review by testgithub-na — COMMENTED** [review]
oke appover to merger with master

File: duck.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+var duck = {
+	name: "abc"
```
Comment (testgithub-na) [inline duck.js:2]: let add prototype for module duck

**Review by testgithub-na — COMMENTED** [review]
please fix code

File: duck.js (hunk 1 above)
Comment (khuatanh) [reply duck.js:2]: oke let me check

**Review by khuatanh — COMMENTED** [review]
ok fix code done

**Pull request merged**